mongodbatlas/data_source_mongodbatlas_federated_settings_identity_providers.go

```diff
@@ -0,0 +1,392 @@
+package mongodbatlas
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
```

Comment:
[No blocking] We should use the same format for imports, here you have matlas without a new line where `mongodbatlas/data_source_mongodbatlas_federated_settings_connected_organizations.go` and `mongodbatlas/data_source_mongodbatlas_federated_settings.go` have the new line

```
	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"

	matlas "go.mongodb.org/atlas/mongodbatlas"
```